sdk/maps/Azure.Maps.Render/src/Models/Options/RenderStaticImageOptions.cs

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+using Azure.Core.GeoJson;
+
+namespace Azure.Maps.Render
+{
+    // cspell:ignore bbox udid
+    /// <summary> Options for rendering static images. </summary>
+    public class RenderStaticImageOptions
+    {
+        /// <summary> Render static image options with bounding box. </summary>
+        /// <param name="boundingBox"> Minimum coordinates (west, south, east, north) of bounding box in latitude longitude coordinate system. </param>
+        public RenderStaticImageOptions(GeoBoundingBox boundingBox)
+        {
+            BoundingBox = boundingBox;
+        }
+
+        /// <summary>
+        /// </summary>
+        public RenderStaticImageOptions(GeoPosition centerCoordinate, int widthInPixels, int heightInPixels)
+        {
+            CenterCoordinate = centerCoordinate;
+            WidthInPixels = widthInPixels;
+            HeightInPixels = heightInPixels;
+        }
+
+        /// <summary>
+        /// Map layer requested. Possible value: <c>StaticMapLayer.Basic</c>, <c>StaticMapLayer.Labels</c> or <c>StaticMapLayer.Hybrid</c>
+        /// If layer is set to <c>StaticMapLayer.Labels</c> or <c>StaticMapLayer.Hybrid</c>, the format should be png.
+        /// </summary>
+        public MapImageLayer? TileLayer { get; set; }
+        /// <summary> Map style to be returned. Possible values are <c>MapImageStyle.Main</c> and <c>MapImageStyle.Dark</c>. </summary>
+        public MapImageStyle? TileStyle { get; set; }
+        /// <summary> Desired zoom level of the map. Zoom value must be in the range: 0-20 (inclusive). Default value is 12. Please see <see href="https://docs.microsoft.com/azure/location-based-services/zoom-levels-and-tile-grid">Zoom Levels and Tile Grid</see> for details. </summary>
+        public int? ZoomLevel { get; set; }
+        /// <summary>
+        /// Coordinates of the center point. Format: 'lon,lat'. Projection used EPSG:3857.
```

Comment:
This all seems very informative but also assumes a good deal of base knowledge.   I don't know that this content is going to be accessible or helpful to developers looking to onboard to the library.